### DISCUSSION TITLE
[Question]  How to use the magnifier

### DISCUSSION THREAD

**Discussion by ffs-akl** [labwc/labwc#2140] [Q&A] [answered]
I've configured the magnifier in my RC.XML,  How to active it ?

```
<magnifier>
      <width>400</width>
      <height>400</height>
      <initScale>2.0</initScale>
      <increment>0.2</increment>
      <useFilter>true</useFilter>
    </magnifier>
```

**Comment by ahesford** [comment, marked as answer]
Configure the actions `ToggleMagnify`, `ZoomIn` and `ZoomOut` described in [labwc-actions(5)](https://labwc.github.io/labwc-actions.5.html#entry_action_name=togglemagnify) to enable or disable the magnifier.

**Comment by ffs-akl** [comment]
Thank you very much!
